lua/astrocommunity/lsp/lsp-signature-nvim/init.lua

```diff
@@ -0,0 +1,9 @@
+return {
+  {
+    "ray-x/lsp_signature.nvim",
+    event = "BufRead",
+    opts = {
+      hint_enable = false, -- disable hints as it will crash in some terminal
+    },
+  },
+}
```

Comment:
```suggestion
return {
  "neovim/nvim-lspconfig",
  dependencies = {
    "ray-x/lsp_signature.nvim",
    opts = {
      hint_enable = false, -- disable hints as it will crash in some terminal
    },
  },
}
```

Should we just lazy load this with `lspconfig`? Since you won't be able to use lsp_signature without LSP